starport/services/serve/plugin-stargate.go

```diff
@@ -84,7 +84,7 @@ func (p *stargatePlugin) PostInit() error {
 	if err != nil {
 		return err
 	}
-	path := filepath.Join(home, p.app.nd(), "config/app.toml")
+	path := filepath.Join(home, fmt.Sprintf(".%s/config/app.toml", p.app.nd()))
```

Comment:
Hey, as I'm aware, Stargate uses non-dot files to keep app data under home. Maybe I'm missing something, have you noticed that if any dot files generated by a Stargate app?

Reply:
oh, I find that it's we changed the directory ourself, I tried with a new test project, no issue exists, I'll close this.